### PR TITLE
Inline math equation

### PR DESCRIPTION
**Pull request by LucasXu0**
### Feature Preview

Screenshot from 2023-07-07 17-51-36

Screenshot from 2023-07-07 17-52-04

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/integration_test/document/document_with_inline_math_equation_test.dart [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      // insert a inline page
+      const formula = 'E = MC ^ 2';
+      await tester.ime.insertText(formula);
+      await tester.editor.updateSelection(
+        Selection.single(path: [0], startOffset: 0, endOffset: formula.length),
+      );
+
+      // tap the inline math equation button
+      final inlineMathEquationButton = find.byTooltip(
+        'Insert an inline math equation',
```
Comment (Xazin) [inline frontend/appflowy_flutter/integration_test/document/document_with_inline_math_equation_test.dart:38]: Not sure why you reverted localization of this?

**Review by Xazin — APPROVED** [review]

File: frontend/appflowy_flutter/integration_test/document/document_with_inline_math_equation_test.dart (hunk 1 above)
Comment (LucasXu0) [reply frontend/appflowy_flutter/integration_test/document/document_with_inline_math_equation_test.dart:38]: The CI failed on generating l10n. I just wanna verify that.

Comment (Xazin) [reply frontend/appflowy_flutter/integration_test/document/document_with_inline_math_equation_test.dart:38]: Alright fair enough

File: frontend/scripts/code_generation/language_files/generate_language_files.sh [hunk 2]
```diff
@@ -10,8 +10,7 @@ cd "$(dirname "$0")"
 # Navigate to the project root
 cd ../../../appflowy_flutter
 
-flutter clean
-
+flutter pub get
 flutter packages pub get
```
Comment (Xazin) [inline frontend/scripts/code_generation/language_files/generate_language_files.sh:14]: These two commands do the same thing

Comment (LucasXu0) [reply frontend/scripts/code_generation/language_files/generate_language_files.sh:14]: correct!. typo.

**Pull request merged**